docs/azure/index-old.yml

```diff
@@ -0,0 +1,201 @@
+### YamlMime:Hub
+
+title: Azure for .NET developers
+summary: Find the information you need to use .NET on Azure
+brand: dotnet
+
+metadata:
+  title: Azure for .NET developers
+  description: Samples, tutorials, and education for .NET on Azure
+  ms.topic: hub-page
+  ms.prod: azure-dotnet
+  ms.date: 03/04/2020
+
+highlightedContent:
+# itemType: architecture | concept | deploy | download | get-started | how-to-guide | learn | overview | quickstart | reference | tutorial | whats-new 
+  items:
+    - itemType: quickstart
+      title: Deploy an ASP.NET web app
```

Comment:
This implies that this guide is just about deploying. Is that the case? 

BTW, I wish the first quickstart was "Build and deploy your first Azure app". It would be a walkthrough through creating an asp.net app, deploying it, adding storage, keyvault, maybe service bus, support for azure monitor, etc.

Reply:
We have some architectural guidance in the .NET docs that may fit the bill. I'll make a note to look.